webapp/src/ts/services/rules-engine.service.ts

```diff
@@ -220,6 +220,7 @@ export class RulesEngineService implements OnDestroy {
     const filterTargetByContext = (target) => target.context ?
       !!this.parseProvider.parse(target.context)({ user: rulesEngineContext.userContactDoc }) : true;
     const targets = settingsTasks.targets && settingsTasks.targets.items || [];
+    const emitter = rulesEngineContext.settingsDoc?.tasks?.disableNools === true ? 'metal' : 'nools';
```

Comment:
This should work for all declarative configurations since https://github.com/medic/cht-conf/pull/215 (required by cht-core v3.8). 

[28 of 49 monitored configurations use declarative as of Apr 2022](https://docs.google.com/spreadsheets/d/1H6ar1yIAvWDKcsNzvxmi9GhZ-0oQAwNBMYA66XAUHG0/edit#gid=928169454).  Configurations of note using nools are Togo and LG. 

Given these stats, I'm unclear how to reliably manage this as the default emitter. Any approach we take to detect nools users would be heuristic in nature. I've tried here to ensure everything is backward compatible by default. I believe we should move toward having this as default, but I'd welcome guidance on how to achieve that.

Reply:
This discussion has [moved here](https://github.com/medic/cht-core/pull/8065#discussion_r1107813970).